catboost/libs/metrics/optimal_const_for_loss.h

```diff
@@ -16,16 +16,24 @@
 
 namespace NCB {
     inline float CalculateWeightedTargetAverage(TConstArrayRef<float> target, TConstArrayRef<float> weights) {
```

Comment:
`boost_from_average` does not work for MultiRMSEWithMissingValues now - it is required to add it near 151 line in this file

Reply:
True. I revert it back to the original implementation.